### ISSUE TITLE
Option to send HTML emails instead of plain text emails

### ISSUE THREAD

**Issue by kgashok** [BlitzKraft/saythanks.io#151]
The note sent to the user should have the option to be sent via email in either text, or HTML format. 

On a related note, https://stackoverflow.com/questions/27050781/python-smtp-emails-showing-both-plain-and-html-versions

**Comment by tarunkumark** [comment]
Sir, I would like to take this issue. I have planned to add radio buttons for plain-text and html and based on the choice, the content type within notify() will change.

**Comment by tarunkumark** [comment]
I have used an email template from [here](https://beefree.io/template/new-years-eve-party-invitation/) to test this feature. For images to work, users will have to use cloud hosted images. I have resorted to not storing these documents in the database as they can sometimes be large and when it is loaded from the DB into your saythanks inbox, it ruins the CSS of your inbox page and you will have to scroll through a huge wall of text to see your next message.
![image](https://user-images.githubusercontent.com/61784876/150677358-41d00e42-53cd-460f-95ba-c4e1eb622ab2.png)
